src/components/Lightbox.js

```diff
@@ -168,7 +168,7 @@ function Lightbox({isAuthTokenRequired, source, onScaleChanged, onPress, onError
             {isContainerLoaded && (
                 <>
                     {isLightboxVisible && (
-                        <View style={[StyleSheet.absoluteFill, {opacity: hasSiblingCarouselItems && isFallbackVisible ? 0 : 1}]}>
+                        <View style={[...StyleUtils.getFullscreenCenteredContentStyles(), {opacity: hasSiblingCarouselItems && isFallbackVisible ? 0 : 1}]}>
```

Comment:
The `opacity` really shouldn't be here and it should be moved to StyleUtils. I know it was existing code, but would you mind moving it?